src/test/java/com/xceptance/xlt/api/util/XltCharBufferTest.java

```diff
@@ -1088,6 +1122,105 @@ public void equalsTest()
         var c2 = XltCharBuffer.valueOf(" foobar--").viewByLength(1, 3);
         assertTrue(c1.equals(c1));
         assertTrue(c2.equals(c1));
+
+        // identical by hashcode, hashcode touched
+        {
+            var s1 = XltCharBuffer.valueOf("foo");
+            var s2 = XltCharBuffer.valueOf("foo");
+            // caches hash codes
+            assertEquals(s1.hashCode(), s2.hashCode());
+            assertTrue(s1.equals(s2));
+            assertTrue(s2.equals(s1));
+        }
```

Comment:
Seems to be the same as the following block. Remove?

Reply:
Removed